src/EntityStore/EntityStore.js

```diff
@@ -72,19 +128,27 @@ export class EntityStore {
       throw new Error(`Specified name is not unique. Name is "${name}"`);
     }
 
+    // todo We can use this.setInjected(), isn't it?
```

Comment:
Nope, we can't. `this.setInjected()` mark current instance as injected, but we use this to mark passed instance (child) as injected.